src/txorphanage.h

```diff
@@ -107,7 +107,7 @@ class TxOrphanage {
     int EraseTxNoLock(const Wtxid& wtxid) EXCLUSIVE_LOCKS_REQUIRED(m_mutex);
 
     /** Timestamp for the next scheduled sweep of expired orphans */
-    NodeSeconds nNextSweep;
+    NodeSeconds n_next_sweep;
```

Comment:
Should be `m_` not `n_`, see https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c